setup.cfg

```diff
@@ -23,7 +23,7 @@ install_requires =
 	pywin32-ctypes!=0.1.0,!=0.1.1; sys_platform=="win32"
 	SecretStorage>=3.2; sys_platform=="linux"
 	jeepney>=0.4.2; sys_platform=="linux"
-	importlib_metadata >= 3.6
+	importlib_metadata < 3.10; python_version < "3.10"
```

Comment:
We should still specify the minimum version of `importlib_metadata`, and I don't see the point of specifying the maximum version (the latest version of backport is 4.x and it should work fine).

Reply:
The correct line would be:
```suggestion
	importlib_metadata >= 3.6; python_version < "3.10"
```

This means: require `importlib_metadata` at least 3.6 for Python older than 3.10.

Reply:
Now please also see this: https://github.com/jaraco/keyring/pull/576#pullrequestreview-987281982

It needs to be replaced in three files: `hook-keyring.backend.py`, `keyring/backend.py`, `tests/test_packaging.py`.